invoices/invoiceregistry_test.go

```diff
@@ -780,6 +781,105 @@ func testKeySend(t *testing.T, keySendEnabled bool) {
 	checkSubscription()
 }
 
+// TestHoldKeySend tests receiving a spontaneous payment that is held.
+func TestHoldKeySend(t *testing.T) {
```

Comment:
a while back i think we settled on `Keysend` vs `KeySend`

Reply:
Yes, happy that you forced us to pick one back then. Fixed.